### PR TITLE
chore: update nvidia-gpu-operator metadata.yaml 

### PR DESCRIPTION
**Pull request by charIesdu**
**What problem does this PR solve?**:

**Which issue(s) does this PR fix?**:
https://d2iq.atlassian.net/browse/D2IQ-92963

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**If the PR adds a version bump, does it add a breaking change in License**:

- [ ] No License Change (or NA).

### CODE REVIEW

File: services/nvidia-gpu-operator/metadata.yaml [hunk 1]
```diff
@@ -11,23 +11,9 @@ certifications:
...
...Ny45NDggMzguODQyYy00LjMxNiAwLTguMTg2LS42OTktMTEuNjY1LTEuODY2VjEzMC4xNWMxNS4zNjQgMS44NTUgMTguNDUyIDguNjQyIDI3LjY4OSAyNC4wMzlsMjAuNTQ0LTE3LjMyM3MtMTQuOTk1LTE5LjY2OC00MC4yNzMtMTkuNjY4Yy0yLjc1MiAwLTUuMzg0LjE5My03Ljk2LjQ3M20wLTQ3LjY3MXYyMS41NTdhOTAuMTY1IDkwLjE2NSAwIDAgMSA0LjI1NS0uMjUzYzU0Ljg4OC0xLjg1IDkwLjY0MSA0NS4wMDcgOTAuNjQxIDQ1LjAwN3MtNDEuMDcxIDQ5Ljk0NS04My44NTkgNDkuOTQ1Yy0zLjkxOSAwLTcuNTkxLS4zNjMtMTEuMDM3LS45NzR2MTMuMzI3YzIuOTQ1LjM3NCA2LjAwNi41OTQgOS4xOTMuNTk0IDM5LjgxNiAwIDY4LjYxMS0yMC4zMzUgOTYuNDk4LTQ0LjQwMSA0LjYxOCAzLjcwNSAyMy41NDQgMTIuNzA1IDI3LjQ0MSAxNi42NTItMjYuNTE2IDIyLjE5NS04OC4zMDIgNDAuMDg2LTEyMy4zMjggNDAuMDg2LTMuMzc0IDAtNi42MjItLjIwNC05LjgwNC0uNTEydjE4LjcyN2gxNTEuMzM3VjcwSDExOS4wOHptMCAxMDMuOTA4djExLjM3M2MtMzYuODMyLTYuNTYyLTQ3LjA1NS00NC44NDctNDcuMDU1LTQ0Ljg0N3MxNy42ODctMTkuNTkxIDQ3LjA1NS0yMi43NjJ2MTIuNDc5Yy0uMDI3IDAtLjAzOC0uMDA2LS4wNjEtLjAwNi0xNS40MDgtMS44NS0yNy40NTIgMTIuNTQ1LTI3LjQ1MiAxMi41NDVzNi43NDkgMjQuMjQzIDI3LjUxMyAzMS4yMTdtLTY1LjQxMy0zNS4xMzFzMjEuODI2LTMyLjIwOCA2NS40MTMtMzUuNTM5di0xMS42OEM3MC44MDMgOTUuNDI3IDI5IDEzNi4zMSAyOSAxMzYuMzFzMjMuNjc2IDY4LjQ1MiA5MC4wOCA3NC43MTZ2LTEyLjQxOWMtNDguNzI4LTYuMTMyLTY1LjQxMy01OS44MzEtNjUuNDEzLTU5LjgzMSIgZmlsbD0iIzc3YjkwMCIvPjwvc3ZnPg==
```
Comment (faiq) [inline services/nvidia-gpu-operator/metadata.yaml:19]: i think you need a new icon

**Review by faiq — APPROVED** [review]

**Comment by faiq** [comment]
looks fine by me

File: services/nvidia-gpu-operator/metadata.yaml (hunk 1 above)
Comment (gracedo) [reply services/nvidia-gpu-operator/metadata.yaml:19]: hmm maybe it's ok to use the same, looks like the gpu operator https://catalog.ngc.nvidia.com/orgs/nvidia/containers/gpu-operator uses the same

image
this is what it is now (copied from old nvidia app)

Comment (gracedo) [reply services/nvidia-gpu-operator/metadata.yaml:19]: image

Unless we wanted the name. I'll ask UX

Comment (gracedo) [reply services/nvidia-gpu-operator/metadata.yaml:19]: synced with UX, leaving it as is

**Review by gracedo — APPROVED** [review]

**Pull request merged**